src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Updated the `ActivitySource` name to the assembly name:
```

Comment:
This is not the assembly name.

Reply:
Right, this should be corrected. But I hope it _is_ consistent with the current guidelines? At least that was the assumption under which I approved it.